### ISSUE TITLE
[T16][Mobile][Accessibility][iPad-3] Unable to attach burning in associated problems in order details

### ISSUE THREAD

**Issue by ranjithswrk** [Mohanme13/Feb-19-Bug-bash#143]
Steps to recreate:
1.Open patient information
2.place order
select associated problems-burning
